specification/redis/resource-manager/Microsoft.Cache/stable/2022-06-01/redis.json

```diff
@@ -2670,12 +2670,12 @@
         "geoReplicatedPrimaryHostName": {
           "type": "string",
           "readOnly": true,
-          "description": "Fully qualified geo subdomain URL which points to geo-primary cache among the linked redis caches."
+          "description": "Fully qualified geo hostname which points to geo-primary cache among the linked redis caches."
```

Comment:
A couple thoughts on the new description. We probably don't need to say 'fully qualified', since that's the default for these internet/cloud computing scenarios, where there is no implicit domain. "geo hostname" also does not seem clear in meaning, can you think of a different term or explanation for this?

Reply:
Updated the description. Please check if it looks better now.